src/ducks/billsMatching/Linker/billsToOperation/findNeighboringOperations.js

```diff
@@ -0,0 +1,90 @@
+const { getDateRangeFromBill, getAmountRangeFromBill } = require('./helpers')
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.